lein-template/src/leiningen/new/duct.clj

```diff
@@ -108,7 +114,14 @@
        (map (fn [[path temp]] [path (render-resource data temp)]))))
 
 (defn generate-project [{:keys [vars templates dirs] :as profile}]
-  (let [data  (merge vars (select-keys profile [:deps :dev-deps]))
+  (let [data (->
+               (merge vars (select-keys profile [:deps :dev-deps
```

Comment:
I think sometihng like this is easier to read:

```clojure
(let [data (-> vars
               (merge (select-keys profile [:deps :dev-deps :repl-options]))
               (assoc :modules      (parse-map modules))
               (assoc :profile-base (parse-map profile-base))
               (assoc :profile-dev  (parse-map profile-dev)))]

Reply:
Do you mean
```clojure
(let [data (-> vars
               (merge (select-keys profile [:deps :dev-deps :repl-options]))
               (assoc :modules      (parse-map (:modules profile)))
               (assoc :profile-base (parse-map (:profile-base profile)))
               (assoc :profile-dev  (parse-map (:profile-dev profile))))]
```
?

Reply:
Yes, but using destructuring instead.